### PR TITLE
PDPDEVTOOL-4021: As a SuiteCloud Developer, I want to have the option to Create SuiteCloud Project in VsCode, so I can build from scratch a new Project using this Code Editor

### PR DESCRIPTION
**Pull request by karl-anthony-ng**

### CODE REVIEW

**Comment by lfpvillegas** [comment]
Could you please change the name of the pull request to the story?

**Review by antoniofruci — APPROVED** [review]

File: packages/vscode-extension/src/commands/CreateProject.ts [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+
+    protected async execute(): Promise<void> {
+        const commandArgs = await this.getCommandArgs();
+        if (Object.keys(commandArgs).length === 0) {
+            return;
+        }
+
+        const actionResult = await this.runSuiteCloudCommand(commandArgs);
+        if (actionResult.isSuccess()) {
+            await CreateProject.openProjectInNewWindow(actionResult.projectDirectory);
```
Comment (lfpvillegas) [inline packages/vscode-extension/src/commands/CreateProject.ts:40]: The story says that there should be a question asking about atttaching the project to the current window or open a new one, but I think is not implemented.

File: packages/vscode-extension/src/commands/CreateProject.ts [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+            if (!overwriteProject) {
+                return {};
+            }
+            commandArgs[COMMAND_ARGUMENTS.OVERWRITE] = overwriteProject === this.translationService.getMessage(ANSWERS.NO) ? 'false' : 'true'; 
+        }
+
+        return commandArgs;
+    }
+
+    private showOverwriteFilesQuestion(projectAbsolutePath: string): Thenable<string | undefined> {
```
Comment (lfpvillegas) [inline packages/vscode-extension/src/commands/CreateProject.ts:127]: In this method you are not only showing the questions but actually getting the result (more exactly the promise, that when using "awaits")
I would name it getOverwriteFiles directly or if you prefer to mantain the "Question" suffix then I would go for "promptOverwriteFilesQuestion" that have the extraction of the response implicit

**Review by lfpvillegas — CHANGES_REQUESTED** [review]

File: packages/vscode-extension/src/commands/CreateProject.ts (hunk 1 above)
Comment (karl-anthony-ng) [reply packages/vscode-extension/src/commands/CreateProject.ts:40]: You are correct there. I had to clarify first with the team before I implemented it. I will be adding that logic here and I'll let you know when I've added it.

File: packages/vscode-extension/src/commands/CreateProject.ts (hunk 2 above)
Comment (karl-anthony-ng) [reply packages/vscode-extension/src/commands/CreateProject.ts]: I updated it and changed itto prompt... as you suggested.

File: packages/vscode-extension/src/commands/CreateProject.ts (hunk 1 above)
Comment (karl-anthony-ng) [reply packages/vscode-extension/src/commands/CreateProject.ts]: Updated to include the logic I mentioned.

**Review by lfpvillegas — APPROVED** [review]

**Review by antoniofruci — APPROVED** [review]

**Pull request merged**